src/rcc.rs

```diff
@@ -1,26 +1,341 @@
 //! Reset and Clock Control
 
-use stm32l4x6::{rcc, RCC};
+// TODO right now the various configure functions reach into rcc directly. This is bad. Add an
+// opaque CR member to RCC, and add methods to CR and BDCR. They should probably take clock source
+// variant arguments.
 
-use ::cmp;
+use stm32l4x6::{rcc, RCC, PWR};
 
 use common::Constrain;
 use time::Hertz;
 use flash::ACR;
 
 impl Constrain<Rcc> for RCC {
+    /// Create an RCC peripheral handle.
+    ///
+    /// Per 6.2, "The MSI is used as system clock source after startup from Reset, configured
```

Comment:
Please instead write along following line.

According to Reference Manual Ch. 6.2:
The default System Clock source is MSI clock with frequency 4 MHz